phpBB/includes/acp/acp_users.php

```diff
@@ -1966,12 +1966,22 @@ function main($id, $mode)
 					$error = $phpbb_avatar_manager->localize_errors($user, $error);
 				}
 
-				$avatar = phpbb_get_user_avatar($user_row, 'USER_AVATAR', true);
+				/** @var \phpbb\avatar\helper $avatar_helper */
+				$avatar_helper = $phpbb_container->get('avatar.helper');
+
+				$avatar = $avatar_helper->get_user_avatar($user_row, 'USER_AVATAR', true);
 
 				$template->assign_vars(array(
 					'S_AVATAR'	=> true,
 					'ERROR'			=> (!empty($error)) ? implode('<br />', $error) : '',
-					'AVATAR'		=> (empty($avatar) ? '<img src="' . $phpbb_admin_path . 'images/no_avatar.gif" alt="" />' : $avatar),
+
+					'AVATAR'			=> empty($avatar['html']) ? '<img src="' . $phpbb_admin_path . 'images/no_avatar.gif" alt="" />' : $avatar['html'],
```

Comment:
there is not to be any html in the backend this needs factored out. You should be instead returning a path to the generic avatar since that is how all avatars will work. In the template we will have img class=o-avatar t-avatar-{{avatar-type}} src={{avatar-src}} alt={{avatar-alt}}